server/services/reportedAdjudicationsService.ts

```diff
@@ -794,7 +794,10 @@ export default class ReportedAdjudicationsService {
     if (!history.length || readOnly) return null
     const finalHistoryItem = history[history.length - 1]
     if (finalHistoryItem.outcome) {
-      if (finalHistoryItem.outcome.outcome.code === OutcomeCode.NOT_PROCEED) {
+      if (
+        finalHistoryItem.outcome.outcome.code === OutcomeCode.NOT_PROCEED &&
+        finalHistoryItem.hearing?.outcome.code !== HearingOutcomeCode.COMPLETE
```

Comment:
Adding an extra check here so that the 'remove hearing' button shows when the hearing is complete and finding is NOT_PROCEED, otherwise it shows the the 'remove outcome' button